lms/templates/main.html

```diff
@@ -158,6 +158,17 @@
     </script>
 % endif
 
+<% gtm_acct = static.get_value("GOOGLE_TAG_MANAGER_ACCOUNT", settings.GOOGLE_TAG_MANAGER_ACCOUNT) %>
```

Comment:
We can also set this value in the [admin panel](/admin/site_configuration/siteconfiguration/1/change/), we just have to add 
...
"GOOGLE_ANALYTICS_ACCOUNT": "value",
"GOOGLE_TAG_MANAGER_ACCOUNT": "value"
}

to the json there. This works OK when we want to have a different configuration for each site.